Formula/pastel.rb

```diff
@@ -0,0 +1,24 @@
+class Pastel < Formula
+  desc "Command-line tool to generate, analyze, convert and manipulate colors"
+  homepage "https://github.com/sharkdp/pastel"
+  url "https://github.com/sharkdp/pastel/archive/v0.5.2.tar.gz"
+  sha256 "9a18110c21e13c43d4b3b2d7b4e6a2b6e06bcf07dc7c95cbaff1e62406c37db7"
+
+  depends_on "rust" => :build
+
+  def install
+    ENV["SHELL_COMPLETIONS_DIR"] = File.join(buildpath, "completions")
+
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    bash_completion.install "completions/pastel.bash" => "pastel.bash"
```

Comment:
Since you're not renaming the file, I think ` => "pastel.bash"` is unnecessary
(same below)